test/loader/serializeInstructionTest.cpp

```diff
@@ -666,4 +666,119 @@ TEST(SerializeInstructionTest, SerializeConstInstruction) {
   };
   EXPECT_EQ(Output, Expected);
 }
+
+TEST(SerializeInstructionTest, SerializeFuncRefControlInstruction) {
```

Comment:
The `br_on_null` and `br_on_non_null` instructions are both control instructions.
Move these into the `SerializeBrControlInstruction` test.